german/the-black-hack.md

```diff
@@ -113,7 +113,7 @@ Zeit im Spiel wird vor allem auf zwei Arten nachgehalten - **Momente** (Runden)
 
 ## Bewegung und Entfernung
 
-Anstatt genaue Werte nachzuhalten, benutzt TBH abstrakte Bereiche, mit denen die Entfernung dargestellt wird. **Benachbart**, **in der Nähe**, **abseits** und **entfernt**. Während eines Zugs können sich Charaktere irgendwo **in der Nähe** bewegen und ihre Handlung jederzeit während der Bewegung durchführen. Sie können ihre Handlung verfallen lassen, um sich **abseits** zu positionieren. Alles jenseits von **abseits** gilt als **entfernt** und würde drei Bewegungen erfordern.
+Anstatt genaue Werte nachzuhalten, benutzt TBH abstrakte Bereiche, mit denen die Entfernung dargestellt wird. **Dicht**, **Nah**, **Weit** und **Fern**. Während eines Zugs können sich Charaktere irgendwo **in der Nähe** bewegen und ihre Handlung jederzeit während der Bewegung durchführen. Sie können ihre Handlung verfallen lassen, um sich **abseits** zu positionieren. Alles jenseits von **abseits** gilt als **entfernt** und würde drei Bewegungen erfordern.
```

Comment:
Ich liebäugle immer noch mit einer Reihe á la "in Reichweite" > "in Schussweite" > "in Sichtweite", aber da fehlt mir immer das 4. Glied in der Kette und ich weiß nicht, ob "in Reichweite" besser zu "Close" oder zu "Nearby" passen würde.

Was ich als "Prüfstein" für Ideen vorschlagen würde, wären die Formulierungen für Zauber, weil da häufig Entfernungen vorkommen und das Original es hinbekommt, jeden Zauber in einer einzigen Zeile zu formulieren. Konkret würde ich "Magic Missile" nehmen, weil da mehrere Entfernungen vorkommen.

***Magic Missile:*** *A Nearby, Far-Away or Distant target takes 1d4 damage/level.*

Bisher:
***Magisches Geschoss:*** *Ein Ziel das in der Nähe, abseits oder entfernt ist, nimmt 1W4 Schaden pro Stufe.*

Ideen:
***Magisches Geschoss:*** *Ein Ziel Nah dran, Weit weg oder Fern ab, nimmt 1W4 Schaden pro Stufe.*
***Magisches Geschoss:*** *Ein Ziel in Reichweite, Schussweite oder Sichtweite, nimmt 1W4 Schaden pro Stufe.*